src/compile_bug_reports.sh

```diff
@@ -143,7 +143,9 @@ process_implementation() {
     header
 
     while IFS= read -r query; do
-        failing_query "$implementation" "$query"
+	if [ -n "${query}" ]; then
```

Comment:
@cburgmer added this condition because sometimes "query" was empty

Reply:
If I understand correctly, this is about `$(all_incorrect_queries)` returning nothing?

Reply:
sometimes "query" is empty, I haven't explored more about it